ngmt/modules/ssd/_pham.py

```diff
@@ -74,7 +77,7 @@ class PhamSittoStandStandtoSitDetection:
     def __init__(
         self,
         cutoff_freq_hz: float = 5.0,
-        accel_convert_unit=9.81,
+        accel_convert_unit = 9.81,
```

Comment:
I don't think that we need to define these `_convert_unit` parameters. I would rather than have a global file that defines constants, like other packages do. 
Also, I am much more a fan of using type hints or not using type hints, but at least being consistent.

Reply:
I handled this issue by eliminating this unit converter since I found out we don't use it inside the algorithm.